test/parallel/test-resource-usage.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const r = process.resourceUsage();
+assert.ok(r.minflt >= 0);
```

Comment:
I'd leave the previous `typeof` assertions in place as well, just to be safe. For example, if by some future mistake these values became numeric strings, these assertions wouldn't catch it.